stdlib/bigarray.mli

```diff
@@ -347,7 +347,7 @@ module Genarray :
      and strictly less than the corresponding dimensions of [a].
      If [a] has Fortran layout, the coordinates must be greater or equal
      than 1 and less or equal than the corresponding dimensions of [a].
-     Raise [Invalid_argument] if the array [a] does not have exactly [N]
+     @raise Invalid_argument if the array [a] does not have exactly [N]
```

Comment:
Ugh, sorry - I just ran the PR through a better diff tool and spotted that this needs to go at the end of documentation comment too!